### DISCUSSION TITLE
Space switcher (Virtual Desktop area like KDE3 used to have)

### DISCUSSION THREAD

**Discussion by StrikeVapors** [lawand-dot-io/taskbar#110] [Feature requests]
A little area with little rectangles that show the existing virtual desktops and smaller rectangles that indicate the windows on those virtual desktops. Clicking on a virtual desktop switches one over to that.

**Comment by researcx** [comment]
And mouse wheel scrolling over it should switch spaces backwards and forwards as it does on Linux.

**Comment by hutber** [comment]
The taskbar is already fantastic, but this is the only thing that I keep moving my mouse to do lol
